package.json

```diff
@@ -150,8 +150,7 @@
     "gulp-sourcemaps": "3.0.0",
     "gulp-svgstore": "9.0.0",
     "gulp-uglify": "3.0.2",
-    "handlebars-helpers": "0.10.0",
```

Comment:
Just flagging there's an [existing PR](https://github.com/uswds/uswds/pull/5599) to remove handlebars-helpers with some additional information. We can go ahead and close that PR in favor of this one 👍